packages/app/next.config.js

```diff
@@ -1,7 +1,26 @@
+const { readdirSync } = require('fs')
+const path = require('path')
+
+const PRESETS_ORIGIN_PATH = path.resolve(__dirname, 'src/components/presets')
+
+const PRESETS = readdirSync(PRESETS_ORIGIN_PATH)
+  .map(filename => path.basename(filename, '.js'))
+  .filter(preset => preset !== 'scope' && preset !== 'index')
```

Comment:
```suggestion
  .filter(!['index', 'scope'].includes(preset))
```

Reply:
ma 2 cents 😛

Reply:
Not applicable here anymore, but yeah I always do this too when checking more than 1 value 👌 . I can do it within generate-previews though.